<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEntry</class>
 <widget class="QDialog" name="ScriptEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Entry</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>521</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Path">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Desc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Path">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_Desc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="box_Type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Type</string>
   </property>
   <widget class="QRadioButton" name="btn_typeBash">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bash</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_typePerl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PERL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_typePython">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Python</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_typePythonGUI">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PythonGUI</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_Parms">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Args</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Args">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbx_Args">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>This script requires or has optional arguments</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbx_ViewMsgs">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View Messages </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbx_ViewCode">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ONLY View Completion Code</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScriptEntry</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScriptEntry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
